bower.json

```diff
@@ -11,11 +11,11 @@
   "license": "Apache License, Version 2.0",
   "repository": {
     "type": "git",
-    "url": "git://github.com/mtscout6/react-router-bootstrap.git"
+    "url": "git://github.com/react-router/react-router-bootstrap.git"
   },
   "dependencies": {
-    "react-router": "^0.11.6",
-    "react-bootstrap": "^0.13.2"
+    "react-bootstrap": ">=0.15",
+    "react-router": ">=0.13.1"
```

Comment:
FWIW, thinking about this a little more - I think the status quo is okay. It's an extra 30 LOC, and the recent classnames major version upgrade means that embedding the dependency will let you avoid conflicts with other packages that specify classnames 1.x.x (since Bower uses flat dependencies rather than nested ones like NPM).

But I agree that an actual Bower user should update this to something they consider sensible.